rest-service/manager_rest/test/endpoints/test_execution_schedules.py

```diff
@@ -147,7 +147,7 @@ def test_schedule_invalid_weekdays(self):
         )
         self.assertRaisesRegex(
             CloudifyClientError,
-            '400: weekdays list contains invalid weekdays',
+            '400: weekdays list contains an invalid weekday',
```

Comment:
plz stop doing this, we have a TON of very tightly coupled checking the error message, and that makes it hell to change them ever, I don't see the point of asserting that the message is exactly this, I say just `"invalid weekday"` is a good regexp for this: it asserts that the error message points to the problem